index.js

```diff
@@ -73,6 +73,12 @@ if (require.main === module) {
           type: 'boolean',
           default: process.env.FORCESSH || false
         },
+        knownhosts: {
+          demand: false,
+          description: 'path to known hosts file',
+          type: 'string',
+          default: process.env.KNOWNHOSTS || '/dev/null',
```

Comment:
would it be better to use `~/.ssh/known_hosts`

Reply:
I was trying to leave the existing functionality intact. I think leaving it as `/dev/null` makes more sense with the way I have strict host checking set. This way, if people want to use `~/.ssh/known_hosts`, they can just provide that option and will have strict host checking enabled.

If it defaults to `~/.ssh/known_hosts`, there will be no way to have strict host checking enabled with that file without adding another flag.